pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -437,8 +438,9 @@ func (s *Service) checkIfLbExists(name string) (*loadbalancers.LoadBalancer, err
 }
 
 func (s *Service) checkIfListenerExists(name string) (*listeners.Listener, error) {
+	mc := metrics.NewMetricPrometheusContext("listener", "list")
```

Comment:
[xref kubernetes/cloud-provider-openstack](https://github.com/kubernetes/cloud-provider-openstack/blob/ab36705fb740299c57cfeeefec6f4185e5f68579/pkg/openstack/loadbalancer.go#L435)
```suggestion
	mc := metrics.NewMetricPrometheusContext("loadbalancer_listener", "list")
```